tap_hubspot/__init__.py

```diff
@@ -1112,7 +1138,10 @@ def do_sync(STATE, catalog):
         except SourceUnavailableException as ex:
             error_message = str(ex).replace(CONFIG['access_token'], 10 * '*')
             LOGGER.error(error_message)
-
+        except UriTooLongException as ex:
+            LOGGER.fatal(f"For stream - {stream.tap_stream_id}, please select less number of properties as "
+                         f"hubspot system doesn't allow huge request uri.")
+            raise ex
```

Comment:
Is there a specific number like Maximum number of fields that can be selected? If Yes, we can mention that number is log message.
`please select 150 or less number of fields`

Reply:
We don't have a specific number of fields. It totally depends on the character length.

Reply:
Done